### PR TITLE
Gives Most High-Risk Items Extended Descriptions

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Gives extended descriptions to most of the high-risk items (excluding the HOS and captain guns, and the blueprints). Also the syndicate krav maga gloves.

The basic examine of several high-risk items has been slightly modified (NAD, blueprints, documents, krav gloves, CMO hypospray, captain's spare, magboots). These changes are flavour only, no critical information has been added or removed.

Note: the HOS and Captain guns received their own descriptions in #25262.

The plant disk cmag interaction has been updated to remain consistent with the NAD's new examine characteristics.

The QM knuckledusters now contain a tiny amount of plasma and titanium (as plastitanium) to explain why they aren't soft and easy to break (gold is a fairly soft metal).

## Why It's Good For The Game
L O R E

## Testing
It compiled.
I examined the items, they had new lore.
Cmagged some plant disks to make sure they worked good.

## Changelog
:cl:
add: Added lore to high risk items.
tweak: Most high risk items have had their descriptions tweaked.
tweak: Plant disk cmag interaction updated to align with new NAD descriptions.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/gene_modder.dm [hunk 1]
```diff
@@ -569,4 +569,20 @@
...
+		return
+	if((user.mind.assigned_role == "Captain" || user.mind.special_role == SPECIAL_ROLE_NUKEOPS) && (user.Adjacent(src)))
+		. += "<span class='warning'>Yes, even closer examination confirms it's not a trick of the light, it really is just a regular plant disk.</span>"
+		. += "<span class='boldwarning'>Now stop staring at this worthless fake and FIND THE REAL ONE!</span>"
+		return
+	. += "Nuclear fission explosives are stored on all Nanotrasen stations in the system so that they may be rapidly destroyed should the need arise."
+	. += ""
+	. += "Naturally, such a destructive capability requires robust safeguards to prevent accidental or mallicious misuse. NT employs two mechanisms: an authorisation code from Central Command, \
+	and the Nuclear Authentication Disk. Whilst the code is normally sufficient, enemies of Nanotrasen with sufficient resources may be able to spoof, steal, or otherwise crack the authorisation code. \
+	The NAD serves to protect against this. It is essentially a one-time pad that functions in tandem with the authorisation code to unlock the detonator of the fission explosive."
```
Comment (1080pCat) [inline code/modules/hydroponics/gene_modder.dm:588]:
```suggestion
	The NAD serves to protect against this. It is essentially a one-time pad that functions in tandem with the authorisation code to unlock the detonator of the fission explosive."

```

File: code/game/objects/items/weapons/cards_ids.dm [hunk 2]
```diff
@@ -638,7 +638,7 @@
 
 /obj/item/card/id/captains_spare
 	name = "captain's spare ID"
-	desc = "The spare ID of the captain."
+	desc = "The spare ID of the captain. Keep this secured"
```
Comment (lewcc) [inline code/game/objects/items/weapons/cards_ids.dm:641]:
```suggestion
	desc = "The spare ID of the captain. Keep this secured."
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 3]
```diff
@@ -668,10 +671,12 @@ GLOBAL_VAR(bomb_set)
...
 	return
 
-//==========DAT FUKKEN DISK===============
+//////////////////////////////
+// MARK: DAT FUKKEN DISK
+//////////////////////////////
 /obj/item/disk/nuclear
-	name = "nuclear authentication disk"
-	desc = "Better keep this safe."
+	name = "\improper Nuclear Authentication Disk"
```
Comment (Contrabang) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:678]: IMO, we should have no capitalization for objects, unless its like X-01 where the capitalization is part of the designation, as its not really a proper noun

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/documents.dm [hunk 4]
```diff
@@ -16,36 +16,56 @@
...
 
 /obj/item/documents/syndicate/red
 	name = "'Red' secret documents"
-	desc = "\"Top Secret\" documents printed on special copy-protected paper. It details sensitive Syndicate operational intelligence. These documents are marked \"Red\"."
+	desc = "Documents printed on special copy-protected paper. They detail sensitive Syndicate operational intelligence."
 	icon_state = "docs_red"
 
+/obj/item/documents/syndicate/red/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='warning'>These documents are marked with \"<b>TOP SECRET - RED</b>\" and the logo of the Syndicate.</span>")
```
Comment (Sirryan2002) [inline code/game/objects/items/documents.dm:36]: same here

File: code/game/objects/items/documents.dm [hunk 5]
```diff
@@ -16,36 +16,56 @@
...
+
 /obj/item/documents/syndicate/blue
 	name = "'Blue' secret documents"
-	desc = "\"Top Secret\" documents printed on special copy-protected paper. It details sensitive Syndicate operational intelligence. These documents are marked \"Blue\"."
+	desc = "Documents printed on special copy-protected paper. They detail sensitive Syndicate operational intelligence."
 	icon_state = "docs_blue"
 
+/obj/item/documents/syndicate/blue/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='warning'>These documents are marked with \"<b>TOP SECRET - BLUE</b>\" and the logo of the Syndicate.</span>")
```
Comment (Sirryan2002) [inline code/game/objects/items/documents.dm:45]: you get my point

File: code/game/objects/items/weapons/defib.dm [hunk 6]
```diff
@@ -377,6 +323,110 @@
...
+	paddle_type = /obj/item/shockpaddles/syndicate
+	combat = TRUE
+	safety = FALSE
+	heart_attack_probability = 100
+
+/obj/item/defibrillator/compact/combat/loaded/Initialize(mapload)
+	. = ..()
+	cell = new(src)
+	update_icon(UPDATE_OVERLAYS)
+/obj/item/shockpaddles/syndicate
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/defib.dm:366]:
```suggestion
	update_icon(UPDATE_OVERLAYS)

/obj/item/shockpaddles/syndicate
```

File: code/modules/clothing/shoes/magboots.dm [hunk 7]
```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/shoes/magboots
 	name = "magboots"
-	desc = "Magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle."
+	desc = "Magnetic boots with a heavy pull, often used during extravehicular activity to ensure the user remains safely attached to the vehicle."
```
Comment (Sirryan2002) [inline code/modules/clothing/shoes/magboots.dm:3]: is this... really necessary? Wouldn't players come to this conclusion that magnetic boots pull really hard just by using them in the first place?

File: code/modules/hydroponics/gene_modder.dm [hunk 8]
```diff
@@ -527,7 +527,7 @@
 /obj/item/disk/plantgene/update_desc()
 	. = ..()
 	if(HAS_TRAIT(src, TRAIT_CMAGGED))
-		desc = "Better keep this safe."
+		desc = "A floppy disk containing 1.44 MB of unique cryptographic identification data. Used along with a valid code to detonate the on-site nuclear fission explosive."
```
Comment (Sirryan2002) [inline code/modules/hydroponics/gene_modder.dm:530]:
```suggestion
		desc = "A floppy disk containing unique cryptographic identification data. Used along with a valid code to detonate the on-site nuclear fission explosive."
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 9]
```diff
@@ -38,9 +38,12 @@ GLOBAL_VAR(bomb_set)
 	/// Is the most recently inputted code correct?
 	var/yes_code = FALSE
 	var/safety = TRUE
+	/// The Nuclear Authentication Disk.
 	var/obj/item/disk/nuclear/auth = null
+	/// The plutonium core.
```
[the remark this answers is not in the document]
Comment (Fordoxia) [reply code/game/gamemodes/nuclear/nuclearbomb.dm:43]: The autodocing shall continue until morale improves.

File: code/game/objects/items/documents.dm [hunk 10]
```diff
@@ -16,36 +16,56 @@
 	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
 
 /obj/item/documents/nanotrasen
-	desc = "\"Top Secret\" Nanotrasen documents printed on special copy-protected paper. It is filled with complex diagrams and lists of names, dates and coordinates."
+	desc = "Nanotrasen documents printed on special copy-protected paper. They are filled with complex diagrams, technical documentation, and lists of names, dates, and coordinates."
 	icon_state = "docs_verified"
 
+/obj/item/documents/nanotrasen/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='warning'>These documents are marked \"<b>TOP SECRET</b> - property of Nanotrasen\".</span>")
```
[the remark this answers is not in the document]
Comment (Fordoxia) [reply code/game/objects/items/documents.dm:24]: Guh! I don't know what was going through my head when I made it like that. I know how to write those properly, I swear! 😓

File: code/modules/clothing/shoes/magboots.dm (hunk 7 above)
Comment (Fordoxia) [reply code/modules/clothing/shoes/magboots.dm:3]: Fair point, I was just transferring that little snippet from the functionally identical syndicate magboots.

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Sirryan2002** [comment]
@Contrabang

**Review by PopeDaveThe3th — APPROVED** [review]
Lore approved.

**Review by lewcc — APPROVED** [review]

File: code/modules/clothing/shoes/magboots.dm [hunk 11]
```diff
@@ -76,20 +76,33 @@
 
 /obj/item/clothing/shoes/magboots/advance
 	name = "advanced magboots"
-	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
```
Comment (Contrabang) [inline code/modules/clothing/shoes/magboots.dm:79]: This feels like a downgrade in terms of a description, from concise to vague and memey

File: code/modules/martial_arts/krav_maga.dm [hunk 12]
```diff
@@ -161,17 +166,35 @@
...
 		style.remove(H)
 
-/obj/item/clothing/gloves/color/black/krav_maga/sec//more obviously named, given to sec
+//////////////////////////////
+// Warden gloves
+//////////////////////////////
+/obj/item/clothing/gloves/color/black/krav_maga/sec
 	name = "Krav Maga gloves"
-	desc = "These gloves can teach you to perform Krav Maga using nanochips."
+	desc = "These gloves can teach you to perform Krav Maga using nanochips (for as long as you're wearing them)."
```
Comment (Contrabang) [inline code/modules/martial_arts/krav_maga.dm:174]: Most descriptions shouldn't have parenthesis
```suggestion
	desc = "These gloves can teach you to perform Krav Maga using nanochips for as long as you're wearing them."
```

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 13]
```diff
@@ -148,6 +149,16 @@
 	. = ..()
 	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
 
+/obj/item/reagent_containers/hypospray/examine_more(mob/user)
```
Comment (Contrabang) [inline code/modules/reagents/reagent_containers/hypospray.dm:152]: Pretty sure you meant this?
```suggestion
/obj/item/reagent_containers/hypospray/CMO/examine_more(mob/user)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/magboots.dm (hunk 11 above)
Comment (Fordoxia) [reply code/modules/clothing/shoes/magboots.dm:79]: Modified.

**Comment by Sirryan2002** [comment]
@Contrabang

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 14]
```diff
@@ -681,11 +686,27 @@ GLOBAL_VAR(bomb_set)
 	/// Is this a training disk?
 	var/training = FALSE
 
+/obj/item/disk/nuclear/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>You should keep this safe...</span>"
```
Comment (Contrabang) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:691]: Danger spans shouldn't be used outside of combat or something happening to the user really. Our descriptions are *mostly* standardized to use only notice and warning.
```suggestion
	. += "<span class='warning'>You should keep this safe...</span>"
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 15]
```diff
@@ -681,11 +686,27 @@ GLOBAL_VAR(bomb_set)
...
+
 /obj/item/disk/nuclear/unrestricted
 	name = "unrestricted nuclear authentication disk"
-	desc = "Seems to have been stripped of its safeties, you better not lose it."
+	desc = "A floppy disk containing unique cryptographic identification data. Used along with a valid code to detonate the on-site nuclear fission explosive."
 	restricted_to_station = FALSE
 
+/obj/item/disk/nuclear/unrestricted/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>This disk has had its safeties removed.</span>"
```
Comment (Contrabang) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:708]: Same here. And perhaps change this to describe what the safeties are.
```suggestion
	. += "<span class='warning'>This disk has had its safeties removed and it will no longer teleport back to the station.</span>"
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 16]
```diff
@@ -681,11 +686,27 @@ GLOBAL_VAR(bomb_set)
...
+	. += "Nuclear fission explosives are stored on all Nanotrasen stations in the system so that they may be rapidly destroyed, should the need arise."
+	. += ""
+	. += "Naturally, such a destructive capability requires robust safeguards to prevent accidental or malicious misuse. NT employs two mechanisms: an authorisation code from Central Command, \
+	and the nuclear authentication disk. Whilst the code is normally sufficient, enemies of Nanotrasen with sufficient resources may be able to spoof, steal, or otherwise crack the authorisation code. \
+	The NAD serves to protect against this. It is essentially a one-time pad that functions in tandem with the authorisation code to unlock the detonator of the fission explosive."
+
 /obj/item/disk/nuclear/unrestricted
 	name = "unrestricted nuclear authentication disk"
-	desc = "Seems to have been stripped of its safeties, you better not lose it."
+	desc = "A floppy disk containing unique cryptographic identification data. Used along with a valid code to detonate the on-site nuclear fission explosive."
```
Comment (Contrabang) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:703]: This is the same description as its parent type, you can remove it
```suggestion
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/nuclear/nuclearbomb.dm (hunk 15 above)
Comment (Fordoxia) [reply code/game/gamemodes/nuclear/nuclearbomb.dm]: All done.

File: code/modules/hydroponics/gene_modder.dm [hunk 17]
```diff
@@ -680,3 +680,21 @@
...
 		. += "<span class='warning'>... Wait. This isn't the nuclear authentication disk! It's a clever forgery!</span>"
+	else
+		. += "<span class='warning'>You should keep this safe...</span>"
+
+/obj/item/disk/plantgene/examine_more(mob/user)
+	. = ..()
+	if(!HAS_TRAIT(src, TRAIT_CMAGGED))
+		return
+
+	if((user.mind.assigned_role == "Captain" || user.mind.special_role == SPECIAL_ROLE_NUKEOPS) && (user.Adjacent(src)))
```
Comment (lewcc) [inline code/modules/hydroponics/gene_modder.dm:691]:
```suggestion
	if((user.mind.assigned_role == "Captain" || user.mind.special_role == SPECIAL_ROLE_NUKEOPS) && user.Adjacent(src))
```

File: code/modules/martial_arts/krav_maga.dm [hunk 18]
```diff
@@ -161,17 +166,35 @@
...
 	item_state = "fightgloves"
 
 /obj/item/clothing/gloves/color/black/krav_maga/sec/Initialize(mapload)
 	. = ..()
 	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
 
-/// for nukies
+/obj/item/clothing/gloves/color/black/krav_maga/sec/examine_more(mob/user)
+	..()
+	. = list()
```
Comment (lewcc) [inline code/modules/martial_arts/krav_maga.dm:184]: oh that's clever. I quite like this as a way to reset things.

Comment (Fordoxia) [reply code/modules/martial_arts/krav_maga.dm:184]: Learned this when working on the energy weapons PR. Avoided the requirement to store the extended descriptions as a var on the item itself whilst still preventing children items from displaying parent descriptions.

File: code/modules/hydroponics/gene_modder.dm [hunk 19]
```diff
@@ -680,3 +680,21 @@
...
+		. += "<span class='warning'>You should keep this safe...</span>"
+
+/obj/item/disk/plantgene/examine_more(mob/user)
+	. = ..()
+	if(!HAS_TRAIT(src, TRAIT_CMAGGED))
+		return
+
+	if((user.mind.assigned_role == "Captain" || user.mind.special_role == SPECIAL_ROLE_NUKEOPS) && user.Adjacent(src))
+		. += "<span class='danger'>Yes, even closer examination confirms it's not a trick of the light, it really is just a regular plant disk.</span>"
+		. += "<span class='userdanger'>Now stop staring at this worthless fake and FIND THE REAL ONE!</span>"
```
Comment (Contrabang) [inline code/modules/hydroponics/gene_modder.dm:693]: I don't like these spans being here, but theres nothing against them, so they can stay.

**Review by Contrabang — CHANGES_REQUESTED** [review]
One last thing. This PR also changes the normal (single-click) description of quite a few high-value items. The PR description only claims that it adds extended (double-click) descriptions. Please ensure that all changes are documented in the PR description.

File: code/game/objects/items/weapons/defib.dm [hunk 20]
```diff
@@ -1,5 +1,14 @@
-//backpack item
-
+/* CONTENTS
```
Comment (FunnyMan3595) [inline code/game/objects/items/weapons/defib.dm:1]: I like the added organization here, but please do it in a separate PR. It's much more difficult to pick out changes when you move code and change it at the same time.

**Review by FunnyMan3595 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/armor_suits.dm [hunk 21]
```diff
@@ -41,6 +44,10 @@
 	item_state = "bulletproof"
 	blood_overlay_type = "armor"
 
+//////////////////////////////
```
Comment (FunnyMan3595) [inline code/modules/clothing/suits/armor_suits.dm:47]: Same problem here as in the defibs: don't move and change in the same PR.

**Review by FunnyMan3595 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/defib.dm (hunk 20 above)
Comment (Fordoxia) [reply code/game/objects/items/weapons/defib.dm]: I've unsorted everything. All the reorganisation is stored on a separate branch for each file for later deployment.

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 22]
```diff
@@ -668,10 +671,12 @@ GLOBAL_VAR(bomb_set)
 				return
 	return
 
-//==========DAT FUKKEN DISK===============
+//////////////////////////////
+// MARK: DAT FUKKEN DISK
+//////////////////////////////
```
Comment (Burzah) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:676]:
```suggestion
// MARK: Nuclear Disk

```

File: code/modules/martial_arts/krav_maga.dm [hunk 23]
```diff
@@ -161,17 +166,35 @@
 	if(H.get_item_by_slot(SLOT_HUD_GLOVES) == src)
 		style.remove(H)
 
-/obj/item/clothing/gloves/color/black/krav_maga/sec//more obviously named, given to sec
+//////////////////////////////
+// Warden gloves
+//////////////////////////////
```
Comment (Burzah) [inline code/modules/martial_arts/krav_maga.dm:171]:
```suggestion
// Warden gloves
```

File: code/modules/martial_arts/krav_maga.dm [hunk 24]
```diff
@@ -135,8 +139,9 @@
 		playsound(D, 'sound/weapons/punchmiss.ogg', 25, TRUE, -1)
 	return TRUE
 
-//Krav Maga Gloves
-
+//////////////////////////////
+// MARK: KRAV MAGA GLOVES
+//////////////////////////////
```
Comment (Burzah) [inline code/modules/martial_arts/krav_maga.dm:144]:
```suggestion
// Krav Maga gloves
```
This is a 200 line file, it does not need a mark.

File: code/modules/martial_arts/krav_maga.dm [hunk 25]
```diff
@@ -161,17 +166,35 @@
...
+	allowing it to direct the wearer's body in the correct manner to execute the selected move when it detects the corresponding activation motion."
+	. += ""
+	. += "Most users are not aware of their body being puppeted in this manner - it feels like a reflexive movement. Particularly observant individuals or unarmed combat specialists trained in a different martial art \
+	do report noticing their limbs being redirected, but they generally get used to it after sufficient exposure."
+	. += ""
+	. += "Because it is the gloves, and not the user that is handling the martial art, users return to previous proficiency when the gloves are removed."
+
+//////////////////////////////
+// Syndicate krav maga gloves
+//////////////////////////////
```
Comment (Burzah) [inline code/modules/martial_arts/krav_maga.dm:197]:
```suggestion
// Syndicate Krav Maga gloves
```

File: code/modules/hydroponics/gene_modder.dm [hunk 26]
```diff
@@ -560,9 +560,9 @@
 	seed.name = "experimental " + seed.name
 	seed.icon_state = "seed-x"
 
-/*
- *  Plant DNA disk
- */
+//////////////////////////////
+// MARK: PLANT DISK
+//////////////////////////////
```
Comment (Burzah) [inline code/modules/hydroponics/gene_modder.dm:565]:
```suggestion
// MARK: Plant Disk

```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 27]
```diff
@@ -763,8 +783,9 @@ GLOBAL_VAR(bomb_set)
 		if(length(open_turfs))
 			return pick(open_turfs)
 
-/// MARK: TRAINING NUKE
-
+//////////////////////////////
+// MARK: TRAINING NUKE
+//////////////////////////////
```
Comment (Burzah) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:788]:
```suggestion
// MARK: Training Nuke

```

**Review by Burzah — CHANGES_REQUESTED** [review]
I want these cleaned up. There is no need for them to be formatted like this, it just adds to the clutter. Mark works just fine, where it can be applied.

File: code/game/gamemodes/nuclear/nuclearbomb.dm (hunk 22 above)
Comment (Fordoxia) [reply code/game/gamemodes/nuclear/nuclearbomb.dm]: All changed.

**Review by Burzah — APPROVED** [review]
